ui/src/app/views/resource-card/component.html

```diff
@@ -43,9 +43,11 @@
                           (receiveDelete)="receiveDelete($event)"></app-confirm-delete-modal>
 
 
-<app-rename-modal [resource]="resourceCard.notebook"
+<app-rename-modal *ngIf="renaming && resourceCard.notebook"
+                  [resource]="resourceCard.notebook"
                   [resourceType]="'Notebook'"
-                  (receiveRename)="receiveNotebookRename($event)"></app-rename-modal>
+                  [onCancel]="cancelRename.bind(this)"
```

Comment:
Note: @panentheos I diverged slightly from what we had in prior PRs - where we bound methods in the constructor and overwrote them: https://github.com/all-of-us/workbench/blob/228888948f45ab8e28ea3160877175775c277caf/ui/src/app/views/homepage/component.ts#L83 Any thoughts on this?

Reply:
That looks good to me, I didn't know you could explicitly reference `this` in the template expression.

Reply:
i like this, more concise